src/app/ReleaseStatusTable.tsx

```diff
@@ -59,7 +59,7 @@ export const ReleaseStatusTable: React.FunctionComponent = () => {
   ];
 
   return (
-    <PageSection isWidthLimited>
+    <PageSection hasBodyWrapper isWidthLimited>
```

Comment:
Just noting that `hasBodyWrapper` should be true by default - https://staging.patternfly.org/components/page#pagesection